server/openapi_server/models/__init__.py

```diff
@@ -3,8 +3,18 @@
 # flake8: noqa
 from __future__ import absolute_import
 # import models into model package
-from openapi_server.models.entity import Entity
+from openapi_server.models.annotation import Annotation
+from openapi_server.models.annotation_source import AnnotationSource
+from openapi_server.models.deidentification_config import DeidentificationConfig
+from openapi_server.models.deidentification_config_deidentification_strategy import DeidentificationConfigDeidentificationStrategy
+from openapi_server.models.deidentification_config_deidentification_strategy_date_offset_config import DeidentificationConfigDeidentificationStrategyDateOffsetConfig
```

Comment:
These variable names are quite long. We could have another look at the API specification of the deidentifier to see if we can simplify these names. Will be done in https://github.com/nlpsandbox/nlpsandbox-schemas/issues/123